src/headingengine.js

```diff
@@ -30,6 +23,39 @@ export default class HeadingEngine extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
+	constructor( editor ) {
+		super( editor );
+
+		const t = editor.t;
+
+		/**
+		 * A set of default localized labels for `config.heading.formats`.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {Object} #_localizedFormatLabels
+		 */
+		const labels = this._localizedFormatLabels = {
```

Comment:
I know :D I thought about the same "solution".

So I guess it's all fine. But this can still be improved in the code:

> User config can be localised together with the default config when they are processed by the _getLocalizedOptions() method (currently _options property, but I commented that it shouldn't be a property). The default config doesn't have to be localized beforehand.